errors.go

```diff
@@ -13,11 +14,22 @@ var (
 )
 
 type taskNotFoundError struct {
-	taskName string
+	taskName   string
+	fuzzyModel *fuzzy.Model
```

Comment:
I think it would make more sense to assign the computed value of `fuzzyModel.SpellCheck(...)` here instead of a more complex struct (errors are meant to be dumb). What do you think?

```suggestion
	didYouMean string
```